xastropy/xguis/abskingui.py

```diff
@@ -44,7 +44,7 @@ class AbsKinGui(QtGui.QDialog):
     """ GUI to analyze absorption lines for future kinematic analysis
     """
     def __init__(self, ispec, z=None, parent=None, llist=None, norm=True,
-                 vmnx=[-300., 300.]*u.km/u.s, abs_sys=None, outfil='dum_kin.json',
+                 vmnx=[-300., 300.]*u.km/u.s, abs_sys=None, outfil=None,
                  sel_wv=None, name=''):
         """
         spec : Filename or Spectrum1D
```

Comment:
Please add docstring for missing parameters and update naming convention when relevant.